sensehat/src/androidTest/java/com/google/android/things/contrib/driver/sensehat/SenseHatDeviceTest.java

```diff
@@ -38,34 +39,38 @@
     @Test
     public void senseHat_DisplayColor() throws IOException {
         // Color the LED matrix.
-        LedMatrix display = SenseHat.openDisplay();
-
-        display.draw(Color.MAGENTA);
-        // Close the display when done.
-        display.close();
+        try (LedMatrix display = SenseHat.openDisplay()) {
+            display.draw(Color.MAGENTA);
+        }
     }
 
     @Test
     public void senseHat_DisplayDrawable() throws IOException {
         Context context = InstrumentationRegistry.getTargetContext();
         // Display a drawable on the LED matrix.
-        LedMatrix display = SenseHat.openDisplay();
-        display.draw(context.getDrawable(android.R.drawable.ic_secure));
-        // Close the display when done.
-        display.close();
+        try (LedMatrix display = SenseHat.openDisplay()) {
+            display.draw(context.getDrawable(android.R.drawable.ic_secure));
+        }
     }
 
     @Test
     public void senseHat_DisplayGradient() throws IOException {
         // Display a gradient on the LED matrix.
-        LedMatrix display = SenseHat.openDisplay();
-        Bitmap bitmap = Bitmap.createBitmap(SenseHat.DISPLAY_WIDTH, SenseHat.DISPLAY_HEIGHT, Bitmap.Config.ARGB_8888);
-        Canvas canvas = new Canvas(bitmap);
-        Paint paint = new Paint();
-        paint.setShader(new RadialGradient(4, 4, 4, Color.RED, Color.BLUE, Shader.TileMode.CLAMP));
-        canvas.drawRect(0, 0, canvas.getWidth(), canvas.getHeight(), paint);
-        display.draw(bitmap);
-        // Close the display when done.
-        display.close();
+        try (LedMatrix display = SenseHat.openDisplay()) {
+            Bitmap bitmap = Bitmap.createBitmap(SenseHat.DISPLAY_WIDTH, SenseHat.DISPLAY_HEIGHT, Bitmap.Config.ARGB_8888);
+            Canvas canvas = new Canvas(bitmap);
+            Paint paint = new Paint();
+            paint.setShader(new RadialGradient(4, 4, 4, Color.RED, Color.BLUE, Shader.TileMode.CLAMP));
+            canvas.drawRect(0, 0, canvas.getWidth(), canvas.getHeight(), paint);
+            display.draw(bitmap);
+        }
+    }
+
+    @Test
+    public void senseHat_BaroTemp() throws IOException {
+        try (BaroTemp baroTemp = SenseHat.openBaroTemp()) {
+            Log.i(BaroTemp.class.getName(),"SenseHat Barotemp raw pressure: "+baroTemp.getBarometerRaw());
```

Comment:
Is there any way we can add assertions to this? Can we verify that the values are non-zero?